resolwe_bio/processes/go_enrichment/go.yml

```diff
@@ -45,16 +44,26 @@
     - name: terms
       label: Enriched terms
       type: basic:json
+    - name: source
```

Comment:
One is input source, the other is output source (for the mapped features). This was requested by @lukaw3d.